shotgun_api3/lib/mockgun.py

```diff
@@ -690,36 +675,91 @@ def _get_field_type(self, entity_type, field):
         except ValueError:
             return self._schema[entity_type][field]["data_type"]["value"]
 
-    def _row_matches_filter(self, entity_type, row, filter):
-        
-        
+    def _row_matches_filter(self, entity_type, row, filter, retired_only):
```

Comment:
I know you didn't add this, but I recommend not using `filter` as a variable name since it is a python [built-in](https://docs.python.org/2/library/functions.html#filter).